src/main/protobuf/v1/events.proto

```diff
@@ -63,4 +63,9 @@ message CatalogItemDescriptorUpdatedV1 {
 
 message MovedAttributesFromEserviceToDescriptorsV1 {
   required CatalogItemV1 catalogItem = 1;
-}
\ No newline at end of file
+}
+
+message CatalogItemRiskAnalysisAddedV1 {
```

Comment:
Our future direction is to manage meaningful events containing the whole domain object.
May I suggest saving the updated CalogItem? 
What do you think @galales ?
PS - The related task describes the solution implemented by @nttdata-rtorsoli.

Reply:
yes, it may be a good idea.
In that case, should we also insert in the event the id of the added risk analysis?
I'm thinking that if we just insert the whole EService it's not possible to know which risk analysis has been added

Reply:
`createdAt` gives you the answer because the risk analysis related to the event is the most recent.
This is not valid for updates.

Reply:
true, but in this context it seems more a hack than a solution.

Since we are going to create new events, IMO it could be a good opportunity to model them as useful as possible.

For example, we could add both the whole parent object, and (when applicable) the id of the impacted sub-object.
That way we could have a unambiguous reference in case of creation, update and deletion.